static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -35,14 +35,19 @@ type IssuesQueryParams = {
   query: string;
 };
 
+const defaultProps = {
+  withChart: false,
+};
+
 type Props = {
   organization: Organization;
   version: string;
   selection: GlobalSelection;
   location: Location;
   defaultStatsPeriod: string;
+  queryFilterDescription?: string;
```

Comment:
since this is optional, should we move it to the end of the object?

Reply:
👍